src/Uno.UWP/Storage/StorageFile.cs

```diff
@@ -34,6 +34,16 @@ private StorageFile(Uri uri)
 			_fileUri = uri;
 		}
 
+		public global::System.DateTimeOffset DateCreated
+		{
+			get
+			{
+				var fileInfo = new FileInfo(Path);
+				return fileInfo.CreationTime;
```

Comment:
Did you ensure it works on all supported platforms?

Reply:
I tested it on Android only - I have no iOS/macOS device.
But it is only simple macro, so as long as FileInfo.CreationTime works, this PR would work also.
Other dates (modified, accessed) are missing on e.g. FAT, but CreationTime should be on every filesystem.
Docs says that this works on Xamarin.Mac, iOS, Android; in .Net Core 1.0 and above, .NET Framework since 1.1, .NET Standard since 1.3. So it should be in everywhere.

Reply:
If it would be more than simple macro, I would add it only for Android, where I can thoroughly test it.

Reply:
I re-created this as #2036, with tests - so, we would know if it works on other platforms too.

Reply:
Why do you recreate PR?  You're loosing the discussion.

Reply:
Because I understand that "one PR = one commit"... Also, I had to recreate my fork of Uno (master in this fork was 'polluted'), so all branches of this fork had to be deleted.
And yes, recreating means loosing discussion (to make some form of continuation, I added reference to old PR), also could lead to some errors in re-created... But I'm not git-master, and even if some other method exist, I don't know it.

Reply:
Now I was corrected that this is not a rule ("one PR = one commit"), but my misunderstanding of comment from Jerome Laban.